### ISSUE TITLE
Terminal pins

### ISSUE THREAD

**Issue by Naujoks212** [DigitalArtDeco/SwitchModePSUinTO3#4]
Hello, what did you use for the terminal pins and where can I buy them? They don't appear on the BOM.

**Comment by DigitalArtDeco** [comment]
Hello, you can use terminal pins like this: 

https://www.mouser.de/ProductDetail/855-H2170-05

BR

**Comment by Naujoks212** [comment]
Thank you, that is excellent.
How about the coil XAL5050-562MEB. That part doesn't seem to be in production anymore and isn't easily available. Is there a suitable replacement?

**Comment by DigitalArtDeco** [comment]
Hello, you can use also the XAL5050-822MEC. This one is still available at mouser. BR

**Comment by Naujoks212** [comment]
I actually received some XAL5050-562MEB, which I found on Aliexpress.
However, the converter is putting out only 1V. I checked all components and I will double check, to make sure I didn't make any mistakes.
I measured the XAL5050, and it only shows 0.01uH, instead of the 8.2uH it's supposed to show.
The parts look genuine. Could it really be that I've been sent fake or damaged parts?

**Comment by DigitalArtDeco** [comment]
Hi, ok you should buy the parts at mouser and not at aliexpress.  You get fake parts for sure.Can you send me some pictures of your work? I am interested ...Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Naujoks212 ***@***.***> Datum: 21.05.24  15:47  (GMT+01:00) An: DigitalArtDeco/SwitchModePSUinTO3 ***@***.***> Cc: Harun Aktas ***@***.***>, Comment ***@***.***> Betreff: Re: [DigitalArtDeco/SwitchModePSUinTO3] Terminal pins (Issue #4) 
I actually received some XAL5050-562MEB, which I found on Aliexpress.
However, the converter is putting out only 1V. I checked all components and I will double check, to make sure I didn't make any mistakes.
I measured the XAL5050, and it only shows 0.01uH, instead of the 8.2uH it's supposed to show.
The parts look genuine. Could it really be that I've been sent fake or damaged parts?

**Comment by Naujoks212** [comment]
![IMG_2962](https://github.com/DigitalArtDeco/SwitchModePSUinTO3/assets/170261868/d0cfe632-c30e-42e3-8e2c-6d06991ddaa4)
Yes, certainly. Here's a picture.

**Comment by Naujoks212** [comment]
Do you maybe have a XAL5050 that I could get from you? I think it might be my problem. To buy even just one from Mouser costs 30€ because of their ridiciulous shipping costs. I'm in Germany as well.

**Comment by Iannic-ann-od** [comment]
> Yes, certainly. Here's a picture.

It looks like you have a small solder blob shorting the top corners of Cinx1 and Cin2.

Also huge TY to the creator for this design!

**Comment by Naujoks212** [comment]
Yes, there is a blob shorting those capactitors. However, these three capactitors are all connected to each other via that big contact on the PCB.

**Comment by DigitalArtDeco** [comment]
> Do you maybe have a XAL5050 that I could get from you? I think it might be my problem. To buy even just one from Mouser costs 30€ because of their ridiciulous shipping costs. I'm in Germany as well.

Hi, yes I have a small stock. How much you need? 
BR

**Comment by Naujoks212** [comment]
Just one, to confirm that my coils are indeed the problem.
Is there any way I can contact you? It doesn't seem possible to send direct messages on Github...

**Comment by DigitalArtDeco** [comment]
> Just one, to confirm that my coils are indeed the problem.
> Is there any way I can contact you? It doesn't seem possible to send direct messages on Github...

Use my official email address "akhar9119@gmail.com ". I will reply to you.  

BR

**Comment by Naujoks212** [comment]
I've sent you an email. Wondering if you received it?
